k8s-infra-variables.yml

```diff
@@ -0,0 +1,37 @@
+---
+- hosts: openstack
+  tasks:
+    - set_fact:
+        cluster_masters:
+          name: kube-masters
+          user: "{{ cluster_deploy_user }}"
+          node_resource: "Cluster::Node"
+          nodenet_resource: "Cluster::NodeNet1"
+          num_nodes: "{{ cluster_num_masters }}"
```

Comment:
This should be fixed at 1 and not able to change. The `kubeadm` role doesn't support multi-master at the moment.